### DISCUSSION TITLE
Question on downgrading

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1056] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
If I downgrade my system, do I have to disable OS verification? I was wondering because I pictured going through all the steps, and then the recovery screen saying: "Your media contains an old ChromeOS! You cannot use it."

**Comment by FunnyCorgi** [comment]
I'm going to get 107 flashed now, and then I will check back soon for other replies.

**Reply by nathblade16** [reply]
check your kver, if its 0 or 1, any version, if its 2, 112-119 I'm pretty sure and if its 3, 120-latest

**Reply by FunnyCorgi** [reply]
kernver ends in a 2

**Reply by FunnyCorgi** [reply]
so you mean I can't downgrade lower than those?

**Reply by FunnyCorgi** [reply]
will downgrading to 112 also downgrade my kernver? This would be useful.
(btw I am keeping my networks caubed until I am able to have my dad get a defender subscription for my windows pc for flashing usb)

**Reply by AshtonDavies** [reply, marked as answer]
@FunnyCorgi No, it does not downgrade your kernel.

**Reply by FunnyCorgi** [reply]
@AshtonDavies Ok thanks, just sorta new to chrome exploits.
